tests/Doctrine/ConnectionFactoryTest.php

```diff
@@ -16,8 +16,8 @@ class ConnectionFactoryTest extends AbstractTest
     /** @var mixed[] $configuration */
     private $configuration = [];
 
-    /** @var mixed[] $connectionFactory */
-    private $connectionFactory = [];
+    /** @var ConnectionFactory $connectionFactory */
+    private $connectionFactory = null;
```

Comment:
```suggestion
    private $connectionFactory;
```

The connection-factory is set in `setUp()`